examples/config/config

```diff
@@ -6,6 +6,7 @@
 # common directory locations
 set prefix      @(echo $UZBL_PREFIX)@
 set data_home   @(echo $XDG_DATA_HOME/uzbl)@
+set data_dirs   @(echo $(echo ${XDG_DATA_DIRS}|sed 's%:%/uzbl:%g')/uzbl)@
```

Comment:
Actually, adding /uzbl to the last directory is the reason why I had to put two nested echo commands.

If XDG_DATA_DIRS=a:b:c, then
echo $(echo ${XDG_DATA_DIRS}|sed 's%:%/uzbl:%g')/uzbl
Will correctly output a/uzbl:b/uzbl:c/uzbl